apps/api/v2/src/modules/auth/guards/billing/platform-plan.guard.ts

```diff
@@ -63,7 +63,7 @@ export class PlatformPlanGuard implements CanActivate {
         canAccess = hasMinimumPlan({
           currentPlan: team.platformBilling?.plan as PlatformPlanType,
           minimumPlan: minimumPlan,
-          plans: ["FREE", "STARTER", "ESSENTIALS", "SCALE", "ENTERPRISE"],
+          plans: ["FREE", "STARTER", "ESSENTIALS", "SCALE", "ENTERPRISE", "CUSTOM"],
```

Comment:
Like Keith mentioned in Slack canva:
<img width="1096" alt="Screenshot 2025-04-09 at 09 06 18" src="https://github.com/user-attachments/assets/8263833e-6dea-432f-8995-f98cb33fc113" />

Why do we call it `CUSTOM` instead of `PER_ACTIVE_USER` or something?